src/pubkey.h

```diff
@@ -7,13 +7,13 @@
 #ifndef BITCOIN_PUBKEY_H
 #define BITCOIN_PUBKEY_H
 
-#include <hash.h>
-#include <serialize.h>
-#include <span.h>
-#include <uint256.h>
+#include <hash.h>       // for Hash, Hash160
+#include <serialize.h>  // for ReadCompactSize, WriteCompactSize
+#include <span.h>       // for Span, MakeSpan
+#include <uint256.h>    // for u256
 
-#include <stdexcept>
-#include <vector>
+#include <cstring>      // for memcmp, memcpy, size_t
+#include <vector>       // for vector
```

Comment:
> Would like feedback on the comments next to the imports.

That is not something we want to do. What is the value-add of putting `// for vector` next to `#include <vector>`? In the other cases, these comments would be fairly unmaintainable, frequently become out of date (if not managed in some automated way), would pollute diffs etc.

Reply:
I agree. Changes have been implemented by removing comments from includes.